app/views/trainees/degrees/edit.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: [ @trainee, @degree ], local: true do |f| %>
+      <h2 class="govuk-heading-l"> Degree Details </h2>
+      <%= f.govuk_error_summary %>
+      <%- degree_subjects = DEGREE_SUBJECTS.map do |name| 
+            OpenStruct.new(name: name) 
+          end 
+          degree_subjects.unshift(OpenStruct.new(name: nil))
+      %>
+      <%= f.govuk_collection_select :degree_subject, degree_subjects, 
+            :name, :name, label: { text: 'Degree subject', size: 's' },
+            hint: { text: 'Search for the closest matching subject' } %>
+
+      <%- institutions = INSTITUTIONS.map do |name| 
```

Comment:
I feel like this should be handled in a helper?

Reply:
next pr @JonF1